app/frontend/styles/_review_warning.scss

```
@@ -0,0 +1,15 @@
+.review-warning {
```

We generally try to use `app-*` for our custom BEM-y CSS modules, as recommended by the Design System:

```suggestion
.app-review-warning {
```

---

Good to know - updated.